sources/3.0.0/sections/10-data_product_format.adoc

```diff
@@ -765,12 +839,27 @@ unsigned +
 The choice of a valid point for the start sequence is determined by the sequencing rule. +
 Format: n, n +
 Example: "0,0" (without quotes)
+
+| 10
+| Vertical datum
+| verticalDatum
+^| 0..1
+<| Integer +
+unsigned +
+16-bit
+| see remark <<tab-root-group-attributes>> +
+row vertical datum
 |===
 
 The gridOriginLongitude, gridOriginLatitude, gridSpacingLongitudinal, and gridSpacingLatitudinal attributes should be in the same geographic units as the bounding box. Note that this practice deviates from S-100 where it indicates that this value should be in Arc Degrees.
 
 numPointsLongitude and numPointsLatitude must contain the number of cells in the x and y dimensions of the values table.
 
+The S-102 uses the "Attribute Overwriting" option of the <<iho-s100,part=10c,clause=9.7.1>>. This allows the feature instance group to overwrite the attributes of a higher group, in this case the 'verticalDatum'. The default vertical datum is specified in the root group (see table <<tab-root-group-attributes>>). A new separate feature instance group must be created for all areas with a vertical datum that differs from the default vertical datum. Only feature instance groups with a value that differs from the default vertical datum use the additional attribute "verticalDatum" on the feature instance group.
+
+[[para-domain-extent-polygon]]
+The use of multiple vertical datums within an S-102 product requires the use of "domainExtent.polygon". In addition, according to S-100, either the BoundingBox at the Feature Instance Group or the "domainExtent.polygon" must be specified. When using multiple vertical datums, the use of the "domainExtent.polygon" is mandatory, which means that the BoundingBox is not specified in this case. The grid cells that do not belong to the area of the respective vertical datum are to be assigned the no-data values. The domainExtent.polgons of different Feature Instance groups must not overlap. At positions where the polygons of different Feature Instance groups touch, the edges must be identical. The domainExtent.polgon does not have to follow the borders of the grid cells, but is an independent vector geometry based on the SoundingDatum surface from S-101. The domainExtent.polgon only supports a simple polygon geometry in accordance with <<iho-s100,part=10c,table=11>>. The mapping of multi-polygons and inner rings is not possible.
```

Comment:
Typos:
`domainExtent.polgons`
`domainExtent.polgon`
Instead of `borders of the grid cells` I'd use `...does not have to follow grid cell boundaries`

Reply:
done in b7d5ead54bb2e04311fbd840167a0a3d19b89a21